awx/ui_next/package.json

```diff
@@ -1,38 +1,66 @@
 {
-  "name": "awx-react",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.jsx",
+  "name": "ui_next",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@lingui/react": "^2.9.1",
+    "@patternfly/patternfly": "^2.71.3",
+    "@patternfly/react-core": "^3.153.3",
+    "@patternfly/react-icons": "^3.15.15",
+    "@patternfly/react-tokens": "^2.8.12",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "ansi-to-html": "^0.6.11",
+    "axios": "^0.18.1",
+    "codemirror": "^5.47.0",
+    "d3": "^5.12.0",
+    "dagre": "^0.8.4",
+    "formik": "^2.1.2",
+    "has-ansi": "^3.0.0",
+    "html-entities": "^1.2.1",
+    "js-yaml": "^3.13.1",
+    "prop-types": "^15.6.2",
+    "react": "^16.13.1",
+    "react-codemirror2": "^6.0.0",
+    "react-dom": "^16.13.1",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "3.4.1",
+    "react-virtualized": "^9.21.1",
+    "rrule": "^2.6.4",
+    "styled-components": "^4.2.0"
+  },
   "scripts": {
-    "start": "webpack-dev-server --config ./webpack.config.js --mode development",
-    "test": "TZ='UTC' jest --coverage",
-    "test-watch": "TZ='UTC' jest --watch",
+    "start": "PORT=3001 HTTPS=true DANGEROUSLY_DISABLE_HOST_CHECK=true react-scripts start",
+    "build": "react-scripts build",
+    "test": "TZ='UTC' react-scripts test --coverage --watchAll=false",
+    "test-watch": "TZ='UTC' react-scripts test",
+    "eject": "react-scripts eject",
     "lint": "eslint --ext .js --ext .jsx .",
     "add-locale": "lingui add-locale",
     "extract-strings": "lingui extract",
     "compile-strings": "lingui compile",
     "prettier": "prettier --write \"src/**/*.{js,jsx,scss}\"",
     "prettier-check": "prettier --check \"src/**/*.{js,jsx,scss}\""
   },
-  "keywords": [],
-  "author": "",
-  "license": "Apache",
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
   "devDependencies": {
-    "@babel/core": "^7.8.3",
-    "@babel/plugin-proposal-class-properties": "^7.1.0",
-    "@babel/polyfill": "^7.0.0",
-    "@babel/preset-env": "^7.8.3",
-    "@babel/preset-react": "^7.0.0",
-    "@lingui/cli": "^2.7.4",
-    "@lingui/macro": "^2.7.2",
+    "@babel/polyfill": "^7.8.7",
+    "@lingui/cli": "^2.9.1",
+    "@lingui/macro": "^2.9.1",
     "@nteract/mockument": "^1.0.4",
```

Comment:
so it seems like we were including mockument before, but I couldn't figure out an equivalent to what is in the test setup file.  what exactly does it do (and does it do anythign different in this PR vs. the way it used to work?

Reply:
mockument adds a polyfill for jsdom. if you comment this line out https://github.com/jakemcdermott/awx/blob/5957-statics/awx/ui_next/src/setupTests.js#L11, you'll see that many of our tests won't work without it. This is why we're using it on devel a well.

Reply:
sounds good, thanks for the explanation